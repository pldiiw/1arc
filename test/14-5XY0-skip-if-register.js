'use strict';

let engine = require('engine').initialize();
const skipIfRegister = require('instructions').skipIfRegister;

engine.data[7] = 143;
engine.data[0xD] = 143;
engine.data[0xE] = 56;
engine.pc = 3049;

const engine1 = skipIfRegister(engine, 7, 0xD);
const engine2 = skipIfRegister(engine1, 7, 0xE);

console.log(engine.pc);
console.log(engine1.pc);
console.log(engine2.pc);
