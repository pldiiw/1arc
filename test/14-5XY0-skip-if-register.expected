3049
3051
3051
